src/Windows/Core/Impl/project.json

```diff
@@ -3,7 +3,8 @@
     "MicroBuild.Core": "0.2.0",
     "Microsoft.Extensions.FileSystemGlobbing": "1.1.0",
     "Microsoft.Extensions.Logging.Abstractions": "1.1.1",
-    "Newtonsoft.Json": "9.0.1"
+    "Newtonsoft.Json": "9.0.1",
+    "System.ValueTuple": "4.3.0"
```

Comment:
Do we need this?

Reply:
Yes, 15.6.4 build fails otherwise. Interface definition from Core references the type.